pkg/scraper/scraper_test.go

```diff
@@ -31,3 +31,12 @@ func TestGetDocument(t *testing.T) {
 	}
 
 }
+
+func TestScrapeJobcenterPageNumber(t *testing.T) {
+
+	_, err := scrapeJobcenterLastPageNumber()
```

Comment:
Unit test should ideally not have external dependencies.

In this case, if jobcentre is down, our test fails.

The right way to do it is to mock the dependency, but it might be too much work to do now so maybe just remove this test when you're done testing for now.